packages/apps/frontera/src/routes/settings/src/components/Tabs/panels/AuthPanel/AuthPanel.tsx

```diff
@@ -121,11 +110,54 @@ export const AuthPanel = observer(() => {
                 />
               )}
               {!store.settings.google.isLoading && (
-                <Switch
-                  isChecked={store.settings.google.gmailEnabled}
-                  onChange={(value) => handleSyncGoogleToggle(value)}
-                  colorScheme='success'
-                />
+                <div className='grid grid-cols-1 gap-3 mt-3'>
+                  {store.settings.google.tokens.map((token) => (
+                    <div className='grid grid-cols-[200px_200px_minmax(100px,_1fr)] gap-2 items-center'>
```

Comment:
Missing key prop here. Key is required by react to reconcile items inside mapped arrays.